### PR TITLE
Pappystein additions

### PR DESCRIPTION
**Pull request by Gordon-Dry**
https://forum.kerbalspaceprogram.com/index.php?/topic/122020-14x-bluedog-design-bureau-stockalike-saturn-apollo-and-more-v142-i-numbered-this-update-before-ksp-updated-1feb2018/&do=findComment&comment=3482502

### CODE REVIEW

**Comment by jsolson** [comment]
Hey @Gordon-Dry if your interested in contributing a little, [this config file](https://github.com/CobaltWolf/Bluedog-Design-Bureau/blob/master/Gamedata/Bluedog_DB/Compatibility/Tweakscale/find_missing.cfg.txt) with it's txt extention removed will identify which parts need Tweakscale added. Knocking out just one or two directories would be appreciated. Last I checked there's about 90 parts in need. It's getting a bit daunting.

File: Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg [hunk 1]
```diff
@@ -44,6 +66,7 @@
         defaultScale = 0.3125
     }
 }
+´
```
Comment (CobaltWolf) [inline Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg]: What is this?

Comment (CobaltWolf) [reply Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg]: @jsolson

**Comment by CobaltWolf** [comment]
I split it up a bit more - made one for the upper solids (should probably rename that config, actually) and made a separate one for Scout.

File: Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg (hunk 1 above)
Comment (jsolson) [reply Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg]: An apostrophe. Just a stray keypress, it's next to the enter key. It can go.

Comment (CobaltWolf) [reply Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg]: It already was gotten rid of when I split the file.

**Comment by Gordon-Dry** [comment]
@jsolson I will do - but beacuse I don't want to use TweakScale I just remove the :NEEDS[TweakScale] for the next session ...

**Comment by Gordon-Dry** [comment]
Just kidding ...

**Comment by Gordon-Dry** [comment]
@jsolson I'm confused - in the configs the pod parts got commented out placeholders.
What do they mean?

Just tell that they're missing?
Or tell that these parts should not be tweakscaleable?

**Comment by jsolson** [comment]
Once upon a time rescaling crewed parts would cause problems. Probably with the interior but I really don't remember. I also don't know if the issues have been addressed. Probably best to leave them commented for now if you can't find out for certain.

Stock procedural fairings also can not be scaled. That may have been fixed by now too, but I don't know for sure.

**Pull request merged**